.github/dependabot.yml

```diff
@@ -4,21 +4,15 @@ updates:
 - package-ecosystem: github-actions
   directory: "/"
   schedule:
-    interval: daily
-    time: '00:00'
-    timezone: UTC
-  open-pull-requests-limit: 10
```

Comment:
Any reason to lift the `open-pull-requests-limit`?

Reply:
We will most likely exceed 10 PRs and we want to make sure that all things get updated.

But I don't think this is an important thing since we will also work on grouping of the PRs. Which makes this irrelevant. And we still have ~20 days to figure this out :)